libtenzir/src/package.cpp

```diff
@@ -387,6 +388,10 @@ auto package_pipeline::parse(std::string_view input)
     }
     rec = *maybe_rec;
     input_.remove_prefix(frontmatter_end + 5);
+    while (not input_.empty()
+           and std::isspace(static_cast<unsigned char>(input_.front()))) {
+      input_.remove_prefix(1);
+    }
```

Comment:
With `#include "tenzir/detail/string.hpp"`

you could write 

```suggestion
    input_ = detail::trim_front(input_);
```

The same applies to the other case.


But feel free to ignore this comment :)